python-sdk/src/astro/table.py

```diff
@@ -132,9 +132,10 @@ def row_count(self) -> Any:
         Return the row count of table.
         """
         db = create_database(self.conn_id)
```

Comment:
Is the `conn_id` always available? For example this is the error i getting as `conn_id` is not available for transform operator
```
  File "/usr/local/airflow/astro_sdk/src/astro/table.py", line 173, in openlineage_dataset_namespace
    database = create_database(self.conn_id)
  File "/usr/local/airflow/astro_sdk/src/astro/databases/__init__.py", line 30, in create_database
    conn_type = BaseHook.get_connection(conn_id).conn_type
  File "/usr/local/lib/python3.9/site-packages/airflow/hooks/base.py", line 70, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
  File "/usr/local/lib/python3.9/site-packages/airflow/models/connection.py", line 432, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `` isn't defined
```

Reply:
@sunank200  does it make sense to have a table object without conn_id? if not then we should check this in the `openlineage_dataset_namespace` method before calling `row_count`

Reply:
https://github.com/astronomer/astro-sdk/pull/1181